### ISSUE TITLE
Status "unbounded" for a correct model?

### ISSUE THREAD

**Issue by hwborchers** [anqif/CVXR#26]
For some benchmark tests I invoked the "catenary example" with `N=1000` chain links and got the following status message:

    N = 1000
    L = 2; h = L/N
    x = Variable(N+1)
    y = Variable(N+1)
    objective  = sum(y)
    constraint = list(x[1] == 0, x[N+1] == 1,
                      y[1] == 1, y[N+1] == 1,
                      diff(x)^2 + diff(y)^2 <= h^2)
    problem <- Problem(Minimize(objective), constraint)
    result <- solve(problem)

    result$getValue(x); result$getValue(y)
    ## [1] NA
    ## [1] NA
    result$status
    ## [1] "unbounded"

The same code with `N=100` returns the usual catenary line with status "optimal", and with `N=500` NAs again and a status of "optimal_inaccurate".

I think, "optimal_inaccurate" is a correct message (the same we get with solver SCS, the result being far off the truth), but a status message "unbounded" for a model that works with a smaller numbers of variables does not seem right.

**Comment by bnaras** [comment]
CVXR is merely reporting what the default solver (ECOS) said. And different solvers report differently. In such cases, using the parameter `verbose = TRUE` to `solve` is revealing.

However, I plan to look into why the version of the problem handed to ECOS by CVXR (and CVXPY too) cause it to fail.

**Comment by bnaras** [comment]
As a note, tweaking the solver parameters seems to help the solver. 
```
result <- solve(problem, FEASTOL = 1e-4, RELTOL = 1e-3, verbose=TRUE)
```
But what I said above still holds.

**Comment by hwborchers** [comment]
Yes, that would be interesting, because when I hand-code the matrices and send them to ECOS resp. SCS directly (without any intervention by CVXR), them both solvers return a good solution within less than 0.5 sec -- no need to set any options.

**Comment by bnaras** [comment]
This is mostly a solver issue, depending on how the problem details are handed to it. This did expose another bug relating to how CVXR expressions are considered positive or negative and that is now addressed in version 0.97+
